distribution/changelog.txt

```diff
@@ -27,6 +27,8 @@
 - Fix: [#8121] Crash Renaming park with server logging enabled.
 - Fix: [#8142] Reliability of mazes and crooked houses can go below 100%.
 - Fix: [#8187] Cannot set land ownership over ride entrances or exits in sandbox mode.
+- Fix: [#8204] Crash when tile element has no surface elements.
+
```

Comment:
Unnecessary whitespace

Reply:
I would have amended this myself, but was on mobile and the editor can't remove lines there.